src/openrct2/ride/TrackDesign.cpp

```diff
@@ -200,15 +200,18 @@ rct_string_id TrackDesign::CreateTrackDesignTrack(const Ride& ride)
         track.type = trackElement.element->AsTrack()->GetTrackType();
 
         uint8_t trackFlags;
-        if (TrackTypeHasSpeedSetting(track.type))
+#ifdef EXPORT_AS_TD6
```

Comment:
It would be better if TrackDesign structs were improved for a NTDF and make T6Import/Export do more work.